Makefile

```diff
@@ -0,0 +1,7 @@
+install:
+	for x in dracut/*; do \
+	  bn=$$(basename $$x); \
+	  install -D -t $(DESTDIR)/usr/lib/dracut/modules.d/$${bn} $$x/*; \
+	done
+	install -D -t $(DESTDIR)/usr/lib/systemd/system systemd/*
+	install -D -t $(DESTDIR)/etc/grub.d grub/*
```

Comment:
Should we still be carrying this?

Reply:
See https://github.com/coreos/ignition-dracut/pull/91
(So currently yes)